src/sentry/tasks/auto_review_inbox.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.models import GroupInbox
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(name="sentry.tasks.auto_review_inbox", time_limit=65, soft_time_limit=60)
+def auto_review_inbox():
+    cutoff = timezone.now() - timedelta(days=7)
+    group_list = list(
+        GroupInbox.objects.filter(date_added__lte=cutoff).values_list("group", flat=True)
```

Comment:
There's also no index on this, so it'll 100% be a full table scan each time. https://github.com/getsentry/sentry/blob/master/src/sentry/models/groupinbox.py#L47

Reply:
Will add an index in its own PR